denote-to-logseq.el

```diff
@@ -0,0 +1,63 @@
+;;; package -- Utilities to migrate denote notes to logseq
+;;
+;; Author: hapst3r
+;; Version: 0.1
+;; URL: https://github.com/bitspook/notes-migrator/denote-to-logseq.el
+;; Keywords: Denote, Logseq
+;; Version: 0.1.0
+;; Package-Requires: ((emacs "27.2") (denote "1.0.0")
+;;
+;;; Commentary:
+;; A WIP package for migrating Denote notes to Logseq
+;;
+;;; Code:
+
+(require 'denote)
+
+(defvar logseq-directory nil)
+
+(defun nm--convert-denote-links-to-logseq ()
+  "Convert all denote links to logseq links in current buffer.
+Current-buffer should be in `org-mode'."
+  (let ((denote-link-rx (rx "[[denote:")))
+    (while (re-search-forward denote-link-rx nil t)
+      (let* ((element (org-element-copy (org-element-context)))
+	     (linked-id (org-element-property :path element))
+	     (linked-file (denote-get-path-by-id linked-id))
+	     (linked-title (denote-retrieve-title-value linked-file 'org))
+	     (logseq-link (format "[[%s]]" linked-title)))
+	(replace-region-contents
+	 (org-element-property :begin element)
+	 (org-element-property :end element)
+	 (lambda () (format "%s " logseq-link)))
+	(org-fill-paragraph)))))
+
+(defun nm--migrate-denote-file-to-logseq (file)
+  "Convert denote note FILE to logseq.
+Copy the file from `denote-directory' to `logseq-directory' and
+convert all denote links to logseq."
+  (let ((dest (expand-file-name
+	       (file-name-nondirectory file)
+               logseq-directory)))
+    (with-temp-buffer
+      (insert-file-contents file)
+      (org-mode)
+      (nm--convert-denote-links-to-logseq)
+      (write-file dest))))
+
+(defun nm--ensure-dir-path (path)
```

Comment:
This function served its purpose when L:40 created destination path of a migrated note by replacing `denote-directory` in note's filepath with `logseq-directory`. Since we are not doing that anymore, I think this function is causing more possible problems while solving none (paths can be tricky to work with across platforms).

In current state of code, I think we should remove this function altogether.